frontend/test/__support__/ui.tsx

```diff
@@ -71,7 +81,15 @@ export function renderWithProviders(
   });
   const history = withRouter ? browserHistory : undefined;
 
-  let reducers = mode === "default" ? mainReducers : publicReducers;
+  let reducers;
+
+  if (mode === "sdk") {
+    reducers = SDK_REDUCERS;
+  } else if (mode === "public") {
+    reducers = publicReducers;
+  } else {
+    reducers = mainReducers;
```

Comment:
nit: should we update the casing for public and main reducers too?

Reply:
I just changed the SDK_REDUCERS  to sdkReducers. Let's conform to what's already there 😄